ci/cloudbuild/builds/lib/features.sh

```diff
@@ -67,3 +67,7 @@ function features::list_full() {
 function features::list_full_cmake() {
   echo "__ga_libraries__,__experimental_libraries__,experimental-storage-grpc,experimental-opentelemetry,experimental-bigquery_rest"
 }
+
+function features::list_doxygen_cmake_no_compute() {
```

Comment:
I think we can exclude `compute` without needing a new function like `list_doxygen_cmake_no_compute`, or a new feature group like `__ga_experimental_no_compute_libraries__`.

I think we can just append `,-compute` to `features::list_full_cmake` in publish-docs.